src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="nav-container">
+        <ul class="nav-container__list">
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav-container__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav-container__item">
+            <a
+              href="#"
+              class="nav-container__link last-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards-container">
+      <article
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__image"></div>
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__text">Product code: 195434</p>
+        <div class="card__stars">
+          <ul class="stars">
```

Comment:
You do not have a modifier for starts.
It is necessary to add a modifier that will be responsible for active stars (filled stars)